docs/docs/guides/airy-core-in-test-env.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Running the Airy Core Platform in a test environment
+sidebar_label: Test
+---
+
+The goal of this document is to provide an overview of how to run the Airy Core
+Platform.
+
+The Airy apps can run on a docker runtime engine, however we recommend deploying the Airy Core Platform inside Kubernetes. For bootstrapping the entire Airy Core Platform on a single machine, we included a [Vagrant](https://www.vagrantup.com) configuration, inside the `infrastructure` directory. All of the services run inside a single Vagrant Kubernetes box.
+
+The Vagrant box is based on Alpine Linux and contains a pre-configured Kubernetes cluster [K3OS](https://k3os.io/), to deploy and run all the Airy Core Platform
+components.
+
+## Getting started
+
+
+To get started please refer to the [bootstrapping](../index.md#bootstrapping-the-airy-core-platform) document.
+
+
+## Debug your installation
+
+You can ssh inside the Airy Core Platform box for testing and debugging purposes with `vagrant ssh` or run commands directly with `vagrant ssh -c COMMAND`
+
+### Status
+
+To view the status of the Vagrant box run:
+
+```sh
+cd infrastructure
+vagrant status
+```
+
+The status command prints the API endpoints, example how to create an user and the public webhooks which are created.
+
+```sh
+cd infrastructure
+vagrant ssh -c /vagrant/scripts/status.sh
+```
+
+### Inspect Kubernetes
+
+```sh
+cd infrastructure
+vagrant ssh
+kubectl get pods
+```
+
+### Start, stop, restart
+You can stop, start or restart the Airy Core Platform box with the following
+commands:
+
+```sh
+cd infrastructure
+vagrant halt
+vagrant up
+vagrant reload
+```
+
+### Re-create the environment
+You can delete and re-create the whole environment with the following commands:
+
+```sh
+cd infrastructure
+vagrant destroy
+vagrant up
+```
+
+
+
+## Access the API
+
+The API services are available by accessing the particular endpoints at http://api.airy/, from your local machine. En example on how to send request to the API is printed by running the `status` command.
+
+## Access the frontend UI
+
+The frontend UI for the demo app can be accessed through http://demo.airy.
+
+The frontend UI for the Airy chat plugin can be accessed through http://chatplugin.airy.
+
+
+## Public webhooks
+
+The public webhook URLs are generated during the bootstrap process and are displayed after the process finishes. You can find your current webhook URLs and your API local address running the `status` command.
+
+## Connect sources
+
+### The Facebook source
+
+To integrate the Facebook source, you must provide your Facebook credentials in the `airy.conf` configuration file, located in the `infrastructure` directory. An example of the configuration can be found in `airy.conf.akk`. After setting the configuration run:
+
+```sh
+vagrant provision --provision-with=conf
```

Comment:
This could result in loading the apps we could add a warning or give a way to set the APP_IMAGE_TAG env variable until we do #396

Reply:
I'll think of a way how to solve this now, maybe instead of using ENV, we can put the variable in the config file?

Reply:
For now, I'll rename the variable to `AIRY_VERSION` and will update the docs.

```
AIRY_VERSION=beta vagrant provision --provision-with=conf
```

Reply:
work for me